src/graphql/schema.js

```diff
@@ -126,6 +177,36 @@ const RootQuery = new GraphQLObjectType({
 const RootMutation = new GraphQLObjectType({
   name: "RootMutationType",
   fields: {
+    addInteractionToContact: {
+      type: InteractionType,
+      args: {
+        contactId: {
+          type: new GraphQLNonNull(GraphQLID),
+        },
+        name: {
+          type: new GraphQLNonNull(GraphQLString),
+        },
+        description: {
+          type: new GraphQLNonNull(GraphQLString),
+        },
+        notes: {
+          type: GraphQLString,
+        },
+        tags: {
+          type: GraphQLString,
```

Comment:
This would need to be a List of Strings... 
```suggestion
          type: new GraphQLList(GraphQLString),
```

Reply:
Woops--error on not adding changes to latest commit. Fixed.